cni/network/network.go

```diff
@@ -139,31 +140,34 @@ func (plugin *NetPlugin) Start(config *common.PluginConfig) error {
 	// Initialize base plugin.
 	err := plugin.Initialize(config)
 	if err != nil {
-		log.Printf("[cni-net] Failed to initialize base plugin, err:%v.", err)
+		log.Logger.Error("[cni-net] Failed to initialize base plugin", zap.Any("error", err))
 		return err
 	}
 
 	// Log platform information.
-	log.Printf("[cni-net] Plugin %v version %v.", plugin.Name, plugin.Version)
-	log.Printf("[cni-net] Running on %v", platform.GetOSInfo())
+	log.Logger.Info("[cni-net] Plugin Info",
+		zap.String("name", plugin.Name),
+		zap.String("version", plugin.Version))
+
+	log.Logger.Info("[cni-net] Os Info", zap.String("platform", platform.GetOSInfo()))
 	platform.PrintDependencyPackageDetails()
 	common.LogNetworkInterfaces()
 
 	// Initialize network manager. rehyrdration not required on reboot for cni plugin
 	err = plugin.nm.Initialize(config, false)
 	if err != nil {
-		log.Printf("[cni-net] Failed to initialize network manager, err:%v.", err)
+		log.Logger.Error("[cni-net] Failed to initialize network manager", zap.Any("error", err))
 		return err
 	}
 
-	log.Printf("[cni-net] Plugin started.")
+	log.Logger.Info("[cni-net] Plugin started")
 
 	return nil
 }
 
 // This function for sending CNI metrics to telemetry service
 func logAndSendEvent(plugin *NetPlugin, msg string) {
-	log.Printf(msg)
+	log.Logger.Info(msg)
```

Comment:
should we change at the place where we format string?

Reply:
That's true, good cactch

Reply:
done